code/modules/client/client_procs.dm

```diff
@@ -285,6 +286,17 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 		else
 			qdel(src)
 			return 0
+	else if (byond_build < ceb) //Out of date client.
+		to_chat(src, "<span class='danger'><b>Your build of BYOND is too old:</b></span>")
+		to_chat(src, CONFIG_GET(string/client_error_message))
+		to_chat(src, "Your build: [byond_build]")
+		to_chat(src, "Required build: [ceb] or later")
+		to_chat(src, "Visit <a href=\"https://secure.byond.com/download\">BYOND's website</a> to get the latest build of BYOND.")
```

Comment:
Merge this code into one for both version and build, gated behind one `if` with a `||`

Players don't know the difference between version and build and copypasted code is bad if it can be avoided.

```
		to_chat(src, "<span class='danger'><b>Your version of BYOND is too old:</b></span>")
		to_chat(src, CONFIG_GET(string/client_error_message))
		to_chat(src, "Your version: [byond_version].[byond_build]")
		if (ceb)
			to_chat(src, "Required version: [cev], revision [ceb] or later") 
		else
			to_chat(src, "Required version: [cev]") 
```